Chapters/1-7-Chapter7.md

```diff
@@ -0,0 +1,493 @@
+
+He took a step closer and I stepped back.
+
+"Think of it this way," he explained. "If something were to harm the you under my watch – the Heiko fidelant's maman, the selfless hero-messenger – it would be disastrous for our entire project. You will surely be in safe hands. A few days on the frontlines of our very species is surely much better than whatever you would get in Ulukoma."
+
+The magistrate butted in, "I believe that Fajro's suggestion would be a fitting alternative and that the state would see more use in applying you to the problem than sending you someplace where you mindlessly twiddle your thumbs in the cold.."
+
+I withdrew from the door and faced the wall. I had never felt so powerless before: my fate was being discussed by a room full of powerful men while I stood in a cell, listening, understanding, with only a choice between two terrible situations. The silence in this room surrounded me, stifled my mind, begged me for an answer. Only the pulse in my head and the gentle shifting of cloth were audible.
+
+"Let's give hae a moment, guys," I heard Kipnu say.
+
+I turned around and found the back of his head and said "Wait, Kipnu. Just you."
+
+The others gathered their lanterns and moved to the other room, the magistrate wrapping his arm around Fajro to help. I walked toward Kipnu but didn't reach for him this time.
+
+"How many people know?" I asked him.
+
+"Know what?"
+
+"What I came here to announce."
+
+"Only the senate. Melna told me it's up to each province to decide that for itself. The governor of Heiko has decided to reveal by hierarchy."
+
+"What's that?"
+
+"In order to stave off panic, people are told one-by-one in private rooms and are given tasks to help so they don't just mentally shut down. My parents were part of the first test, back when we had to ration for the long nights of ash."
+
+"I remember learning about that from a friend. I was too young to remember and my parents never wanted to talk about it."
+
+"All the farmers were working their hardest for the few days we had until the ash swallowed the sky. Then they cursed the air and covered their houses like the rest of us. They got the same rations as the rest of us of course, but you could see that they were crushed."
+
+"We are a proud bunch. When the steam lines and electric rails were coming up, we held on to tradition. There's a certain sense with crops that farmers have. No machine could ever copy it."
+
+"I beliieve it. I've seen it myself."
+
+I reached for his shoulders under a thin, tightly woven tunic that he usually wore under his medical aprons. It reached for comfort more than for attention.
+
+"Kipnu... What do you know about Fajro?" I asked.
+
+"That man who was just here?"
+
+"Yes. I met him on the train today."
+
+"I've never seen him before."
+
+"What? But he's a patient of yours?"
+
+"I've never met that man in my life."
```

Comment:
😱  shock again!

Reply:
How does he know so much about them?

Reply:
He's a politician who likely had to get intelligence about his rival fidelants, including Melna. He would naturally know all about Melna's personal life as well.